paulo-bc/MetadataHDWalletKit
# Xcode 15 support

## Objective

Fix a crash on launch on iOS 16.x when app is built with Xcode 15.0.x

## Description

* [fix] add `@_optimize(none)` to a `Data` extension in `Data+Script.swift`
* removed couple of unused scripts/support files
* update git ignore
* add example project (xcodegen)
* bump min ios and macos version in Package file
* increase min secp256k1 and CryptoSwift versions in Package file

## How to test

* Generate project, open it, change scheme config to Release, run on a iOS 16.x sim, app should open and show hello world.
* Run tests in both iOS 16 and iOS 17 sims, they should all pass.
